src/Mimi.jl

```diff
@@ -379,6 +380,28 @@ function getindex(mi::ModelInstance, component::Symbol, name::Symbol)
     return getfield(mi.components[component].Variables, name)
 end
 
+""" returns the size of index i in model m"""
+function getindexcount(m::Model, i::Symbol)
+    return m.indices_counts[i]
+end
+
+""" returns the values of index i in model m"""
+function getindexvalues(m::Model, i::Symbol)
+    return m.indices_values[i]
+end
+
+""" returns the index labels of the variable or parameter in the given component"""
+function getindexlabels(m::Model, component::Symbol, x::Symbol)
+    metacomp = getmetainfo(m,component)
```

Comment:
@davidanthoff @jrising I changed it to just use the word "index". However for variables and parameters originally I called it "getdimensions" because the metacomp.variables and metacomp.parameters attributes are called dimensions. Should that be changed too or is there a reason that they were originally called dimensions instead of indices?